fea-rs/src/compile/variations.rs

```diff
@@ -17,32 +15,29 @@ use write_fonts::{
 /// This trait abstracts over that info.
 pub trait VariationInfo {
     /// If the tag is an axis in this font, return the min/max values from fvar
-    fn axis_info(&self, axis_tag: Tag) -> Option<AxisInfo>;
-
-    /// Return the normalized value for a user coordinate for the given axis.
-    ///
-    /// NOTE: This is only used for computing the normalized values for ConditionSets.
-    ///
-    /// NOTE: if unit suffixes become a thing, and ConditionSets remain a thing, then
-    /// this should use the same AxisLocation enum as below.
-    fn normalize_coordinate(&self, axis_tag: Tag, value: Fixed) -> F2Dot14;
```

Comment:
this is still something that we would need for the proposed variable syntax extension for FEA, specifically to support feature variations (it's unrelated to variable value records etc)

Reply:
It's done slightly differently now, for example https://github.com/googlefonts/fontc/blob/2cfc562e421647628ac5669fca9c62fae805adbb/fea-rs/src/compile/compile_ctx.rs#L1143-L1151 supports all types of location and converts them to normalized. If we need that in several places we can hoist it out. I will leave such hoisting for future as I dont' currently have a second location that needs it.